packages/angular/tests/e2e/ngmodel.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+
+test('Form component ngModel', async ({ page }) => {
+  await page.goto('http://localhost:4200/forms');
+
```

Comment:
Let's add initial state tests here, usually it's just to verify that the values are indeed blank before the fille / update happens.

Something like...
```
  await expect(page.locator('#input-name')).toHaveText('');
```

etc. Each form component would need their own initial state test.